test/extended/util/disruption/disruption.go

```diff
@@ -187,6 +187,7 @@ func (cma *chaosMonkeyAdapter) Test(sem *chaosmonkey.Semaphore) {
 	cma.test.Setup(cma.framework)
 	defer cma.test.Teardown(cma.framework)
 	ready()
+	defer g.GinkgoRecover()
```

Comment:
I wouldn't expect `GinkgoRecover` in places other than just the body of the Ginkgo test, ie. inside `It`, `Describe`, `Before*` etc.

Reply:
That was also my assumption, but I was still seeing panics and wanted to see if adding a recover as close to the test would help.

Reply:
You should add those in the tests where you see those failures, if they don't have it, yet.

Reply:
I think all of the panics are all coming from the cluster upgrade test. I added a recover statement at the top of that test[1], but I don't think this solved the problem completely. Could the upgrade test spawn additional goroutines which would also need the recover call?

[1] https://github.com/openshift/origin/pull/26307/files#diff-a3e17408eaaa387d9a91030c6d3cd0fe5ad10f976a605dc1d081b56a57f79162R134